### ISSUE TITLE
Azure AD - 'new sign in experience'

### ISSUE THREAD

**Issue by jyjohnson** [aws-azure-login/aws-azure-login#13]
Hi - previously the new login UI was not the default. I think MS is now making it the default (not 100% sure about that - maybe random).

Getting this error:
aws-azure-login Populating and submitting login form +2m
info: TypeError: undefined is not an object (evaluating 'document.forms[0].login.value = username')
info:   undefined:2 in anonymous
info:   :6

because input names have changed to this:
`<div aria-hidden="true" style="cursor:text" data-bind="text: hintText, css: hintCss" class="placeholder">Email or phone</div>`

the next page has this:
```
<input name="passwd" type="password" id="i0118" autocomplete="off" class="form-control" aria-describedby="passwordError loginHeader passwordDesc" aria-required="true" data-bind="
                    textInput: password,
                    hasFocusEx: isFocused,
                    placeholder: $placeholderText,
                    ariaLabel: str['CT_PWD_STR_PwdTB_AriaLabel'],
                    css: { 'has-error': error }" placeholder="Password" aria-label="Enter password">
```

Thanks.

**Comment by marcvi** [comment]
Hi,

Experiencing exactly the same problem here.

**Comment by dtjohnson** [comment]
Yeah, I see it. Will get a fix out...

**Comment by dtjohnson** [comment]
I just published v0.6.3 with a *hack* to navigate to the old login experience. This should get this working again while I sort out a longer term solution.

**Comment by marcvi** [comment]
Hi,
It does work for me, awesome.
Many thanks.

**Comment by dtjohnson** [comment]
It's a temporary fix. I assume Microsoft will force the switch at some point.

**Comment by jyjohnson** [comment]
Thanks!

**Comment by dtjohnson** [comment]
I just published v1.0.0 with a long term fix for the new login page.
